### PR TITLE
Run3-sim119 Provide overlap checking cfg's for 2026 geometries: DDD and DD4hep

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Provide overlap checking cfg's for 2026 geometries: DDD and DD4hep

#### PR validation:

Use the cfg's to find overlaps in the 2026D92 scenario

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

File: SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck2026DDD_cfg.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Way to use this:
 #   cmsRun g4OverlapCheck2026_cfg.py geometry=D86 tol=0.1
```
Comment (perrotta) [inline SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck2026DDD_cfg.py:3]: Perhaps for a privately used config this update of a comment line is not necessary:
```suggestion
#   cmsRun g4OverlapCheck2026DDD_cfg.py geometry=D86 tol=0.1
```

@bsunanda please let us know if you want to update it, or if you agree merging the PR as it is, instead

Comment (civanch) [reply SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck2026DDD_cfg.py:3]: @perrotta , I would guess DD4hep is our default for these checks, not DDD, but Sunanda may clarify.

Comment (perrotta) [reply SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck2026DDD_cfg.py:3]: > @perrotta , I would guess DD4hep is our default for these checks, not DDD, but Sunanda may clarify.

@civanch tbat's not the point. The point is that in the instruction to run it the name of the file is wrong. This is a very minor issue for a script that has to be run privately, by experts I imagine. In any case, if @bsunanda agrees we can even fix it and merge without retriggering tests and signature, as it is just a comment line

**Comment by bsunanda** [comment]
For phase2 I believe it is still DDD. DD4Hep have some issues which are not addressed as yet. We should try to address them soon.
________________________________

**Comment by bsunanda** [comment]
I shall change the name in the comment line
________________________________
From: Sunanda Banerjee
Sent: 17 June 2022 14:56
To: cms-sw/cmssw; cms-sw/cmssw
Cc: Mention
Subject: RE: [cms-sw/cmssw] Run3-sim119 Provide overlap checking cfg's for 2026 geometries: DDD and DD4hep (PR #38401)

For phase2 I believe it is still DDD. DD4Hep have some issues which are not addressed as yet. We should try to address them soon.
________________________________

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by perrotta** [comment]
please abort
(don't re-run tests for just a change in a comment line)

**Pull request merged**